server/views/pages/debug/reportDetails.njk

```diff
@@ -2,138 +2,246 @@
-      }
-    ],
-    [
-      {
-        text: "Incident time"
-      },
-      {
-        text: report.incidentDateAndTime | dateAndTime
-      }
-    ],
-    [
-      {
-        text: "When reported"
+    {
+      key: {
+        text: "Date and time of incident"
       },
-      {
-        text: report.reportedAt | dateAndTime
+      value: {
+        html: report.incidentDateAndTime | dateAndTime
       }
-    ],
-    [
-      {
-        text: "Reported by"
+    },
+    {
+      key: {
+        text: "Description"
       },
-      {
-        text: usersLookup[report.reportedBy].name or report.reportedBy
+      value: {
+        html: report.description
       }
-    ],
-    [
-      {
-        text: "Description"
+    }
+  ]
+}) }}
+
+<!-- Incident type table -->
+{{ govukSummaryList({
+  card: {
+    title: {
+      text: "Incident type"
+    },
+    actions: {
+      items: [
+        {
+        href: "#",
+        text: "Change",
+        visuallyHiddenText: "incident type"
+        }
+      ]
+    }
+  },
+  rows: [
+    {
+      key: {
+        text: "Incident type"
       },
-      {
-        text: report.description
+      value: {
+        html: typesLookup[report.type] or report.type
       }
-    ]
+    }
   ]
-  }) }}
+}) }}
 
-  <h2>Prisoners involved</h2>
+<!-- Prisoners involved table -->
   {%- if report.prisonersInvolved.length > 0 -%}
-    <ul>
+    {%- set prisonerRows = [] -%}
       {%- for prisonerInvolved in report.prisonersInvolved -%}
-        <li>
-          <a href="{{ dpsUrl }}/prisoner/{{ prisonerInvolved.prisonerNumber }}">
-            {% if prisonersLookup[prisonerInvolved.prisonerNumber] %}
-              {{ prisonersLookup[prisonerInvolved.prisonerNumber] | nameOfPerson }}
-            {% else %}
-              {{ prisonerInvolved.prisonerNumber }}
-            {% endif %}
-          </a>
-          ({{ prisonerInvolved.prisonerRole }}) -
-          {% if prisonerInvolved.outcome %}
-            Outcome: {{ prisonerInvolved.outcome }}
-          {% else %}
-            No outcome
-          {% endif %}
-        </li>
+        {% if prisonersLookup[prisonerInvolved.prisonerNumber] %}
+          {%- set prisonerName = prisonersLookup[prisonerInvolved.prisonerNumber] | nameOfPerson -%}
+        {% else %}
+          {%- set prisonerName = prisonerInvolved.prisonerNumber -%}
+        {% endif %}
+
+        {%- set prisonerRole = prisonerInvolvementLookup[prisonerInvolved.prisonerRole] or prisonerInvolved.prisonerRole -%}
+        {% if prisonerInvolved.outcome %}
+          {%- set outcome = prisonerOutcomeLookup[prisonerInvolved.outcome] or prisonerInvolved.outcome -%}
+        {% else %}
+          {%- set outcome = 'No outcome' -%}
+        {% endif %}
+
+        {%- set prisonerURL = dpsUrl + '/prisoner/' + prisonerInvolved.prisonerNumber -%}
+        {%- set prisonerRows = (prisonerRows.push({
+          key: {
+            html: '<a href=' + prisonerURL + ' class="govuk-link">' + prisonerName + '</a>'
```

Comment:
this block would be a little better to do in a slightly better way. for instance, `prisonerName` is not escaped and the href attribute won't have quotes. you could write the whole html block outside this with:

```nunjucks
{% set prisonerURLWhatever %}
  <a href="{{ prisonerURL }}">{{ prisonerName }}</a>
{% endset %}
```
so that this last line becomes `html: prisonerURLWhatever`